internal/commands/result.go

```diff
@@ -414,31 +423,53 @@ func SummaryReport(
 	} else if summary.TotalIssues == 0 {
 		summary.RiskMsg = "No Risk"
 	}
-	return summary, nil
 }
 
-func countResult(summary *wrappers.ResultSummary, result *wrappers.ScanResult) {
-	engineType := strings.TrimSpace(result.Type)
-	if contains(summary.EnginesEnabled, engineType) && isExploitable(result.State) {
-		if engineType == commonParams.SastType {
-			summary.SastIssues++
-			summary.TotalIssues++
-		} else if engineType == commonParams.ScaType {
-			summary.ScaIssues++
-			summary.TotalIssues++
-		} else if engineType == commonParams.KicsType {
-			summary.KicsIssues++
-			summary.TotalIssues++
-		}
-		severity := strings.ToLower(result.Severity)
-		if severity == highLabel {
-			summary.HighIssues++
-		} else if severity == lowLabel {
-			summary.LowIssues++
-		} else if severity == mediumLabel {
-			summary.MediumIssues++
-		} else if severity == infoLabel {
-			summary.InfoIssues++
+func enhanceWithScanSummary(summary *wrappers.ResultSummary, resultsWrapper wrappers.ResultsWrapper) error {
+	scanSummary, errModel, err := resultsWrapper.GetScanSummariesByScanIDS(map[string]string{
+		"scan-ids": summary.ScanID,
+	})
+	if err != nil {
+		logger.PrintIfVerbose(err.Error())
+		return err
+	}
+	if errModel != nil {
+		logger.PrintIfVerbose(errModel.Message)
+		return errors.Errorf("%s: CODE: %d, %s", failedGettingScan, errModel.Code, errModel.Message)
+	}
+
+	if scanSummary == nil || len(scanSummary.ScansSummaries) > 1 {
```

Comment:
We should maybe check if the length is not equal to 1, in case it comes empty line 445 will throw.